src/plugins/guided_onboarding/common/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { GuideState } from '@kbn/guided-onboarding';
```

Comment:
Can you remind me again why `GuideState` lives in a package?

Reply:
We needed `GuideState` in the Home plugin and then moved some of the code from the Home plugin into a package, so the types have to be in the package too.